doc/bluechi-examples/golang/README.md

```diff
@@ -0,0 +1,35 @@
+# golang examples
+
+First, make sure the system has golang installed and download the dbus library.  
+
+These steps were based on CentOS Stream 9:
+
+**Installing go**:
+
+``` bash
+# dnf install go-toolset
+```
+
+**Adding dbus library**:
+
+``` bash
+# go install github.com/godbus/dbus
+```
+
```

Comment:
Could you add another section here for showing the content of `list_nodes.go`? Then you can integrate the source file like this
```go
--8<-- "api-examples/go/list_nodes.go"
```

Maybe we need to add a new package to the readthedocs, but we can do that after this PR is merged.